reana_job_controller/utils.py

```diff
@@ -72,6 +71,8 @@ def initialize_krb5_token(workflow_uuid):
 class SSHClient:
     """SSH Client."""
 
+    import paramiko
```

Comment:
If we import `paramiko` in this class, which is a good thing to do, we have to amend all paramiko-related commands to use `self.paramiko` as the package name, for example:

```diff
diff --git a/reana_job_controller/utils.py b/reana_job_controller/utils.py
index 9176495..418ba7a 100644
--- a/reana_job_controller/utils.py
+++ b/reana_job_controller/utils.py
@@ -81,8 +81,8 @@ class SSHClient:

     def establish_connection(self):
         """Establish the connection."""
-        self.ssh_client = paramiko.SSHClient()
-        self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        self.ssh_client = self.paramiko.SSHClient()
+        self.ssh_client.set_missing_host_key_policy(self.paramiko.AutoAddPolicy())
         self.ssh_client.connect(hostname=self.hostname, port=self.port, gss_auth=True)

     def exec_command(self, command):

```

Reply:
Done 👍